recipes/default.rb

```diff
@@ -31,19 +34,25 @@
   'a9d9e6308931fa2a2b0cada070516d45b76d752430c31c9198933c78f8d54b17'
 
 # rvm package depends
-%w(sqlite-devel libyaml-devel readline-devel zlib-devel libffi-devel
```

Comment:
Why was this changed?

Reply:
There are a lot of missing packages on rhel8 I was hoping to ge  you to audit the packages I should include. I am not sure what is needed

Reply:
Next time it would be ideal to have a comment here explaining that.

Reply:
sorry